locale/pt-br/index.md

```diff
@@ -11,8 +11,8 @@ labels:
   tagline-lts: Recomendado Para Maioria dos Usuários
   changelog: Changelog
   api: Documentação da API
-  version-schedule-prompt: Ou verifique o
-  version-schedule-prompt-link-text: Cronograma de versões LTS
+  version-schedule-prompt: For information about supported releases, see the
```

Comment:
```suggestion
  version-schedule-prompt: Para mais informações sobre o suporte das versões, consulte o 
  version-schedule-prompt-link-text: cronograma de lançamentos
```

Reply:
Remove the duplicated "version-schedule-prompt-link-text" in English.